src/app/fund.model.ts

```diff
@@ -1,17 +1,17 @@
 /**
  * Only Champion Fundings are used in the app thus far, not Pledges.
  */
-export class Fund {
-  constructor(
+export type Fund = {
```

Comment:
we were never calling the constructor, never actually making any object that has this as its prototype. We only used the class as a compile time type. Hence changing it to a type to better reflect reality.